src/webview/js/__tests__/rewriteHtml.js

```diff
@@ -1,7 +1,7 @@
 /** @jest-environment jsdom-global */
 // @flow strict-local
 
-import rewriteImageUrls from '../rewriteImageUrls';
```

Comment:
fa7f7539a webview [nfc]: Refactor rewriteImageUrls to rewriteHtml.

Commit message nits: "Rename", "create".

Reply:
I thought this form ("Renames") of commit message body is the one recommended by Zulip [Docs](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)?

>The body is written in prose, with full paragraphs; each paragraph should be separated from the next by a single blank line.

Reply:
Ah, I think the mobile project diverges from the server project in this respect. We acknowledge that the "what" of a commit message may overflow the summary line, into the body, and we prefer keeping the same syntactic form (often called the imperative; I'm on the fence about whether it's kind of a base infinitive, it would be good to figure that out 😆). @gnprice has posted some thoughts about this at https://github.com/zulip/zulip-mobile/pull/4079#issuecomment-622188734.

Reply:
Note that as described in that comment, there's variation on this within the server project :slightly_smiling_face: -- some of us consistently use the same syntactic form as in the summary line, others consistently use the indicative (like "Renames"), and Tim varies between the two. But in the mobile repo we've generally been consistent in using the same form as in the summary line.